src/constants/token/pancakeswap.json

```diff
@@ -1090,6 +1090,22 @@
       "chainId": 56,
       "decimals": 18,
       "logoURI": "https://exchange.pancakeswap.finance/images/coins/0x42712dF5009c20fee340B245b510c0395896cF6e.png"
+    },
+    {
+      "name": "Gourmet Galaxy",
```

Comment:
BEP20 contract does not have a space in the `name` field, but I guess it's fine.